notebooks/01_data_on_manifolds.ipynb

```diff
@@ -0,0 +1,669 @@
+{
```

Comment:
<p>I think you can add ";" at the end of <code>ax.legend()</code>  to avoid <code>&lt;matplotlib.legend.Legend at 0x12268cdd0&gt;</code>  from printing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/786/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Yes, thanks!